### PR TITLE
Export data from Movimiento table.

### PR DESCRIPTION
**Pull request by Juerodriguez**
## Resumen

 Este PR permite descargar todos los archivos de la tabla movimiento en un zip
Corresponde a la issue: https://github.com/PyAr/OpenLex/issues/10.

## Checklist

- [x] Variables, funciones y comentarios en Español
- [x] Corrector ortográfico ok
- [x] Cdigo Python cumple PEP8
- [x] Cobertura de tests con Playwright
- [x] Revisores asignados (pares & al menos 1 mentor)

## Screenshots

Captura de pantalla de 2021-06-23 13-12-02

https://user-images.githubusercontent.com/73370773/123356902-fc267700-d53e-11eb-957a-c004c64155a3.mp4

https://user-images.githubusercontent.com/73370773/123356913-ff216780-d53e-11eb-90dc-4ecc8e95b012.mp4

### CODE REVIEW

**Comment by Juerodriguez** [comment]
Cambie el formato de videos de playwright a mp4 para hacer posible la subida de los mismos a github

File: controllers/expedientes.py [hunk 1]
```diff
@@ -64,6 +64,32 @@ def index():
     return locals()
 
 
+@auth.requires_login()
+def download():
+    import io
```
Comment (eamanu) [inline controllers/expedientes.py:69]: Import io at the top of the file

File: controllers/expedientes.py [hunk 2]
```diff
@@ -64,6 +64,32 @@ def index():
...
+    zip_filename = 'Movimiento.zip'
```
Comment (eamanu) [inline controllers/expedientes.py:70]: Could be a good Idea set the zip file name ('Movimientos.zip') in 
a 'global' variable, thinking that in the future we can set this in a
configuration file.

File: controllers/expedientes.py [hunk 3]
```diff
@@ -64,6 +64,32 @@ def index():
...
+    tempfile = io.BytesIO()
+    temparchive = zipfile.ZipFile(tempfile, 'w', zipfile.ZIP_DEFLATED)
+    rows = db(db.movimiento.archivo != None).select()
+    try:
+        for file_id in rows:
+            file_single = file_id.archivo
+            if not file_single:
```
Comment (eamanu) [inline controllers/expedientes.py:77]: I don't remember if I propose this change, but looking it now
we might don't negate here and write all the logic into the `if`
I mean:
```python
    if file_single:
        file_loc = db.movimientos.archivo.retrieve_file_properties ....
        file_name = db.movimientos.archivo.retrieve_file_properties ....
        temparchive.write...
```

File: controllers/expedientes.py [hunk 4]
```diff
@@ -64,6 +64,32 @@ def index():
...
+            file_loc = db.movimiento.archivo.retrieve_file_properties(file_single)['path']+ '/' + file_single
+            file_name = db.movimiento.archivo.retrieve_file_properties(file_single)['filename']
+            temparchive.write(file_loc, file_name)
+    finally:
+        temparchive.close()
+        tempfile.seek(0)
+
+    response.headers['Content-Type'] = 'application/zip'
+    response.headers['Content-Disposition'] = 'attachment; filename = %s' % zip_filename
+    res = response.stream(tempfile, chunk_size=4096)
```
Comment (eamanu) [inline controllers/expedientes.py:89]: `chunk_size` also can be in a global variable.

**Comment by eamanu** [comment]
@Juerodriguez also check that tests are failling https://github.com/PyAr/OpenLex/pull/36/checks?check_run_id=2911412429

Let me know if you need help

**Comment by Juerodriguez** [comment]
> @Juerodriguez also check that tests are failling https://github.com/PyAr/OpenLex/pull/36/checks?check_run_id=2911412429
> 
> Let me know if you need help

I could see that this dont work because I forget commit one line change.
Now I upload that with suggest changes

File: controllers/expedientes.py [hunk 5]
```diff
@@ -81,11 +106,15 @@ def vista_expediente():
         user_signature=True)
     links = [A('Expediente', _href=url, _type='button',
                _class='btn btn-default')]
-    for k in linked_tables:
+    for k in LINKED_TABLES:
```
Comment (eamanu) [inline controllers/expedientes.py:109]: :1st_place_medal:

**Review by eamanu — APPROVED** [review]

**Review by NicolasSandoval — APPROVED** [review]
LGTM!

**Pull request merged**